elytron/src/main/java/org/wildfly/extension/elytron/PolicyDefinitions.java

```diff
@@ -146,6 +146,9 @@ static ResourceDefinition getPolicy() {
             @Override
             protected void performRuntime(OperationContext context, ModelNode operation, ModelNode model) throws OperationFailedException {
                 String defaultPolicy = context.getCurrentAddress().getLastElement().getValue();
+                if (model.hasDefined(ElytronDescriptionConstants.DEFAULT_POLICY)) {
+                    defaultPolicy = ElytronExtension.asStringIfDefined(context, DEFAULT_POLICY, model);
+                }
```

Comment:
From the code I am pretty sure default value should be removed.
(btw, it works, because ElytronExtension.asStringIfDefined, as opposite to DEFAULT_POLICY.resolveModelAttribute, does not resolve default value and returns undefined, default value is ignored)